### PR TITLE
Pgadmin: 6.21 -> 7.0

### PR DESCRIPTION
**Pull request by gador**
###### Description of changes

Update of Pgadmin to current version 7 Changelog: https://www.pgadmin.org/docs/pgadmin4/latest/release_notes_7_0.html

Includes updates to
- [python3Packages:azure-mgmt-resource: 22.0.0 -> 23.0.0](https://github.com/NixOS/nixpkgs/commit/51c492376deeef087ac16214c61075027bcf47c1)
- [python3Packages.flask-socketio: 5.1.1 -> 5.3.3](https://github.com/NixOS/nixpkgs/commit/937302555a942ab258b214f89248898493cb43d8)

This is a major release and enables us to remove all of the custom overrides. I decided to still keep the scope "umbrella" in case the next minor update will require an older version of some python package.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gador** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 226675` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
22 packages built:
</summary>
<ul>
<li>appdaemon</li>
<li>appdaemon.dist</li>
<li>azure-cli</li>
<li>azure-cli.dist</li>
<li>gdbgui</li>
<li>gdbgui.dist</li>
<li>onionshare</li>
<li>onionshare-gui</li>
<li>onionshare-gui.dist</li>
<li>onionshare.dist</li>
<li>pgadmin4</li>
<li>pgadmin4-desktopmode</li>
<li>pgadmin4-desktopmode.dist</li>
<li>pgadmin4.dist</li>
<li>python310Packages.azure-mgmt-resource</li>
<li>python310Packages.azure-mgmt-resource.dist</li>
<li>python310Packages.flask-socketio</li>
<li>python310Packages.flask-socketio.dist</li>
<li>python311Packages.azure-mgmt-resource</li>
<li>python311Packages.azure-mgmt-resource.dist</li>
<li>python311Packages.flask-socketio</li>
<li>python311Packages.flask-socketio.dist</li>
</ul>
</details>

File: pkgs/development/python-modules/flask-socketio/default.nix [hunk 1]
```diff
@@ -26,6 +30,7 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     coverage
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flask-socketio/default.nix:31]:
```suggestion
```
We don't want to generate coverage

**Comment by bendlas** [comment]
I've got issues after this update:

The web interface tries to access `/static/js/generated/*` files, which lead to 404 errors. I believe ther is a `yarn run bundle` step missing.

**Comment by gador** [comment]
@bendlas  thank you for report! 

I believe #229184 should fix the issue.

**Pull request merged**